scripts/terraform/aws/m3-etcd/main.tf

```diff
@@ -0,0 +1,67 @@
+# durable network interfaces
+resource "aws_network_interface" "m3-etcd" {
+  subnet_id       = "${var.subnet_id}"
+  security_groups = ["${var.sg_ids}"]
+
+  tags {
+    Application          = "m3"
+    Name                 = "${var.instance_name}"
+    configuration-source = "${var.config_source}"
+  }
+}
+
+# public elastic IPs
+resource "aws_eip" "m3-etcd" {
```

Comment:
[ETCD across regions need to be tuned to handle the timing delays for leadership election](https://github.com/etcd-io/etcd/blob/master/Documentation/faq.md#does-etcd-work-in-cross-region-or-cross-data-center-deployments)